packages/cli-upload/src/upload.js

```diff
@@ -87,6 +88,10 @@ export const upload = command('upload', {
 
   const tokenType = percy.client.tokenType();
 
+  if (!ALLOWED_TOKEN_TYPES.includes(tokenType)) {
+    exit(1, 'Invalid Token Type, only web, self managed token type are allowed.');
```

Comment:
we don't exit here, we'll throw `error`, thats what current `exec` behaviour is.

we need to stop percy local server. so L130 should be executed somehow.

Reply:
Moved the check before starting percy (no matching files handled similarly)